packages/storybook/src/avatar/css-avatar.stories.tsx

```diff
@@ -0,0 +1,167 @@
+/* @license CC0-1.0 */
+
+import readme from '@kernteam/components-css/avatar/README.md?raw';
+import type { Meta, StoryObj } from '@storybook/react';
+import { IconBuilding, IconUser } from '@tabler/icons-react';
+import { CSSProperties, PropsWithChildren } from 'react';
+import '@kernteam/components-css/avatar/index.scss';
+
+interface AvatarProps {
+  borderRadius?: string;
+  backgroundColor?: string;
+  color?: string;
+  imageSrc?: string;
+  altText?: string;
+  privateProfile?: boolean;
+}
+
+const Avatar = ({
+  borderRadius,
+  backgroundColor,
+  color,
+  imageSrc,
+  altText,
+  privateProfile,
+  children,
+}: PropsWithChildren<AvatarProps>) => {
+  return (
+    <button
+      className="kernteam-avatar"
+      style={
+        {
+          '--kernteam-avatar-border-radius': borderRadius,
+          '--kernteam-avatar-background-color': backgroundColor,
+          '--kernteam-avatar-color': color,
+        } as CSSProperties
```

Comment:
`as CSSProperties` is niet nodig, je kunt dit in een `custom-css-properties.d.ts` file zetten:

```ts
import 'react';

declare module 'react' {
  export interface CSSProperties {
    [key: `--kernteam-${string}`]: string | number | undefined;
  }
}
```